style.css

```diff
@@ -1,340 +1,926 @@
+}
+
+.catalog__title {
+    position: relative;
+    padding-left: 30px;
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    order: 1;
+}
+
+.catalog__title::before {
+    position: absolute;
+    content: '';
+    top: 3px;
+    left: 0px;
+    width: 24px;
+    height: 24px;
+    background-image: url('images/folder.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.catalog__navigation:after {
+    order: 2;
+    content: '';
+    background: #D0D7FB2B;
+    height: 1px;
+    flex: 1 1 auto;
+}
+
+.catalog__item {
+    padding: 30px;
+    margin-right: 24px;
+    background-color: #1D1C28;
+    transition: all 2s ease;
+}
+
+.catalog__item:hover {
+    background-image: url(images/hover-up-left.png), url(images/hover-down-right.png);            
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: top left, bottom right;
+    transition: all 2s ease;
+}
+
+.catalog__item--hover {
+    background-color: #1D1C28;
+}
+
+.catalog__image {
+    position: relative;
+}
+
+.catalog__price {
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+}
+
+.catalog__name {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    color: #EFF5FA;
+}
+
+/* Index page Blog block */
+
+.blog__wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.blog__post {
+    display: grid;
+    grid-row: span 2;
+    padding: 30px;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    background-color: #1D1C28;
+}
+
+.blog__image {
+    position: relative;
+}
+
+.blog__image img {
+    width: 100%;
+    object-fit: cover;
+}
+
+.blog__post--small {
+    height: 435px;
+}
+
+.blog__post--small .blog__image {
+    overflow: hidden;
+}
+
+.blog__info {
+    display: flex;
+}
+
+.blog__name,
+.blog__date {
+    position: relative;
+    padding-left: 20px;
+}
+
+.blog__name {
+    padding-right: 36px;
+}
+
+.blog__name::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/user.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 3px;
+    left: 0px;
+    width: 16px;
+    height: 16px;
+}
+
+.blog__name:after {
+    content: '';
+    position: absolute;
+    top: 4px;
+    right: 18px;
+    width: 2px;
+    height: 16px;
+    background-color: #D0D7FB2B;
+}
+
+.blog__date::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/calendar.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 3px;
+    left: 0px;
+    width: 16px;
+    height: 16px;;
+}
+
+/* Index page Our Team block */
+
+.team {
+    max-width: 1300px;
+    margin-inline: auto;
+}
+
+.team__members {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.team__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
Delete:
display: flex;
flex-direction: column;
align-items: center;
and
margin: 0 30px 30px 0;
and
height: 293px;

Reply:
done